pom.xml

```diff
@@ -201,10 +201,12 @@
     </dependencies>
```

Comment:
@bttanakagn 
It is better here.
* `Selenium Test` -> `End to End Test`.
* Or, `Unit Test` -> `Test`, and remove `Selenium Test`.